### PR TITLE
fix (abr-testing): Checks that most recent run is completed in abr_scale script

### PR DESCRIPTION
**Pull request by rclarke0**
# Overview

Checks that run being pulled is completed. Adds error handling if no new runs need to be added.

# Test Plan

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

File: abr-testing/abr_testing/data_collection/abr_google_drive.py [hunk 1]
```diff
@@ -141,6 +149,11 @@ def create_data_dictionary(
                 continue
     transposed_runs_and_robots = list(map(list, zip(*runs_and_robots)))
     transposed_runs_and_lpc = list(map(list, zip(*runs_and_lpc)))
+    try:
+        num_of_runs_added = len(runs_and_robots)
+        print(f"{num_of_runs_added} run(s) added.")
+    except UnboundLocalError:
```
Comment (sfoster1) [inline abr-testing/abr_testing/data_collection/abr_google_drive.py]: this is usually not an exception you want to be catching. Maybe make `runs_and_robots` optionally `None` instead and always exist?

**Review by sfoster1 — CHANGES_REQUESTED** [review]
Spidey-sense tingles at that UnboundLocalError, I think that should be optionally `None` instead

File: abr-testing/abr_testing/data_collection/abr_google_drive.py (hunk 1 above)
Comment (rclarke0) [reply abr-testing/abr_testing/data_collection/abr_google_drive.py]: runs_and_robots is defined at the beginning so i think i can get rid of the error handling

**Review by sfoster1 — APPROVED** [review]
Looks good!

**Pull request merged**